client/js/lounge.js

```diff
@@ -1039,9 +1042,7 @@ $(function() {
 			socket.emit("names", {target: self.data("id")});
 		}
 
-		if (screen.width > 768 && chan.hasClass("chan")) {
-			input.focus();
-		}
+		focus();
```

Comment:
On current master clicking the cycle button closes the keyboard for me, this PR does a neat trick by triggering click, then focusing on the input to keep the keyboard open. I can do the same for the send button.